Networking-in-Android-Lab/app/src/main/java/com/ezequielc/networking_in_android_lab/MainActivity.java

```diff
@@ -0,0 +1,125 @@
+package com.ezequielc.networking_in_android_lab;
+
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+    public static final String CEREAL_URL = "http://api.walmartlabs.com/v1/search?query=cereal&format=json&apiKey=k6u2srf6huv44a45a2pyu9w6";
+    public static final String CHOCOLATE_URL = "http://api.walmartlabs.com/v1/search?query=chocolate&format=json&apiKey=k6u2srf6huv44a45a2pyu9w6";
+    public static final String TEA_URL = "http://api.walmartlabs.com/v1/search?query=tea&format=json&apiKey=k6u2srf6huv44a45a2pyu9w6";
```

Comment:
Thank you for using constants! Just to make sure you know, you can compose constants out of other constants, so rather than repeat a nearly identical URL each time you could do
```java
private static final String BASE_URL = "http://api.walmartlabs.com/v1/search?&format=json&apiKey=k6u2srf6huv44a45a2pyu9w6&query=";
private static final String CEREAL_URL = BASE_URL + "cereal";
private static final String CHOCOLATE_URL = BASE_URL + "chocolate";
private static final String TEA_URL = BASE_URL + "tea";
```